Tribler/community/tunnel/routing.py

```diff
@@ -95,7 +96,14 @@ def tunnel_data(self, destination, payload):
         @return bool: whether the tunnel request has succeeded, this is in no
          way an acknowledgement of delivery!
         """
-        return self.proxy.tunnel_data_to_end(destination, payload, self)
+
+        self._logger.error("Tunnel data (len %d) to end for circuit %s with ultimate destination %s", len(payload),
```

Comment:
I know, I left it at error for now to show @lfdversluis know that there are a huge amount of small packets being sent.